src/Tizen.Network.Smartcard/Tizen.Network.Smartcard/SmartcardSession.cs

```diff
@@ -211,7 +211,8 @@ public void CloseChannels()
         /// <exception cref="InvalidOperationException">Thrown when the method failed due to invalid operation.</exception>
         public SmartcardChannel OpenBasicChannel(byte[] aid, byte p2)
         {
-            int ret = Interop.Smartcard.Session.SessionOpenBasicChannel(_sessionHandle, aid, aid.Length, p2, out _basicChannel);
+            int aidLen = (aid == null ? 0 : aid.Length);
+            int ret = Interop.Smartcard.Session.SessionOpenBasicChannel(_sessionHandle, aid, aidLen, p2, out _basicChannel);
```

Comment:
Is it meaningful to pass a `null` value into `Session OpenBase Channel` API?

Reply:
Yes, If the Application ID (AID) is null, it means no Applet is to be selected on this channel and the "default Applet" will be used.